_posts/2017-08-05-MusED-Hackathon-2017.md

```diff
@@ -0,0 +1,21 @@
+---
+title:  "MusED Hackathon 2017"
+date:   2017-09-04 12:00:00 -0400
+categories: update
+---
+The [MusED Hackathon]({{site.baseurl}}{% link _events/2017-08-05.md %}), held on August 5, revolved around Music Education, Music Experience Design, and musing about and muses for Music. It was held in collaboration with the [IMPACT Conference](https://impact.musedlab.org/) and NYU Steinhardt's [MusEDLab](https://musedlab.org/) lab.
+
+After kicking off the day with a set of invigorating talks at 12 pm and a pizza break before 2 pm, participants took part in workshops or gathered in groups and created hacks.
+
+![Img: Thoughts from the talks](/assets/events/20170805/20170805_talk4.jpg)
+
+At 8pm, hacking ended. Many participants presented their work from the day. To view the final presentations, watch the twitter video below. All projects related to the theme are displayed on the [event page]({{site.baseurl}}{% link _events/2017-08-05.md %}) and under [projects]({{site.baseurl}}{% link projects.md %}).
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="und" dir="ltr"><a href="https://t.co/8ugmU6fYxX">https://t.co/8ugmU6fYxX</a></p>&mdash; Music Hackathon (@musichackathon) <a href="https://twitter.com/musichackathon/status/893988613867937793">August 6, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Kushagra Sharma, Manaswi Mishra, and Siddharth Bhardwaj, visiting us from [Music Hackathons Bulgaria](https://www.facebook.com/MusicHackathonsBG/) kicked off the presentations with the project that brough them to NY, Samsara. They presented that, as well as the project they worked on at the hackathon, and were followed by Todd Gregory, Derek Bioh, Liam Baum, Patrick S. Lee, Adam November, Cole Mahlowitz, Jonathan Marmor, Antony Fonville, Tasdique, Sammy, Omar, and Aman. The projects ranged from apps (using music and visualizations and in support of performing artists), to cardboard and aluminum instruments, to a physical version of a beat generation app, to a musical composition created from a video recording.
+
+![Img: Final Presentation Audience](/assets/events/20170805/20170805_finalPresentations.jpg)
+
+We thank all the participants, speakers, volunteers, and our co-sponsors for helping make this a wonderful event.
```

Comment:
It seems like several of the images aren't used. Was that your intention?

Reply:
Yes! I couldn't figure out how to create a gallery/tile the images using markdown, or flip through them in place, so decided that perhaps a better way to use them would be to include them in the individual project writeups. I plan on adding them there. 

Do you think it makes more sense to just dump them all at the end of this post?